### ISSUE TITLE
Error in Braille with liblouis

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#4960]
**Reported by jjimenez0 on 2015-03-06 07:05**
Hi,
Sometimes i have a fail in NVDA related with Braille and lib louis. I don't know the conditions in order to reproduce it, but i need to know if the problem is with lib louis or maybe problem with the driver of my braille display. 
We are developing a driver for the EcoBraille displays in NVDA (you can see it in ticket #4078) and it is working fine, but we have this bug sometimes.
I attach a log with the error we have and a dump of memory, just in case.
I will be very grateful if you can say which is the best way to solve this problem. Thanks in advance
Juan Ramón

**Comment by jcsteh** [comment]
We believe this to be the same underlying cause as #4457. A fix is now available for testing in the latest "next" branch snapshots.
